image_provisioner/playbooks/roles/docker-config/tasks/main.yaml

```diff
@@ -5,29 +5,39 @@
     state: enforcing
```

Comment:
The SELinux task will happen either way, but it seems that `docker_storage_driver` must be set one way or another (overlay or devicemapper). There is no default.